GliaWidgets/SecureConversations/SecureConversations.PendingInteraction.swift

```diff
@@ -0,0 +1,79 @@
+import Combine
+
+extension SecureConversations {
+    final class PendingInteraction: ObservableObject {
+        @Published private(set) var pendingStatus = false
+        @Published private(set) var unreadMessageCount = 0
+        @Published private(set) var hasPendingInteraction = false
+        let environment: Environment
```

Comment:
I see that you don't use `environment` property directly. Based on that, does it make sense to make it `private`?

Reply:
Makes sense 1e8962cc29e69274952b0867a44adb75771438d2.